Neos.ContentGraph.PostgreSQLAdapter/src/Domain/Projection/NodeRecords.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Neos.ContentGraph.PostgreSQLAdapter package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\ContentGraph\PostgreSQLAdapter\Domain\Projection;
+
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * The collection of node records
+ *
+ * @implements \IteratorAggregate<int,NodeRecord>
```

Comment:
@internal

Reply:
so is the whole directory... we should either use it everywhere or nowhere imho